.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,56 @@
+# This workflow will deploy the application to the running environment after a successful merge to main
+
+name: deploy
+
+on:
+    workflow_run:
+        workflows: [test] # Triggered by the test workflow
+        types: [completed] # The test workflow must complete
+        branches: [main] # Only when main is updated
+
+jobs:
+    on-success:
+        runs-on: ubuntu-latest
+        if: ${{ github.event.workflow_run.conclusion == 'success' }} # Only when test completed successfully 
+        steps:
+            - uses: actions/checkout@v2
+            - name: Install dependencies
+              run: |
+                python -m pip install --upgrade pip
+                pip install wheel
+                pip install -r requirements.txt
+                pip install -e .
+      
+            - name: Prepare FlaskBB
+              run: |
+                flaskbb translations compile
+
+            - name: Build binary distribution and upload to artifactory
+              run: zip -r "flaskbb_deploy-${{github.sha}}.zip" ./flaskbb ./.ebextensions wsgi.py setup.py setup.cfg requirements.txt flaskbb.cfg celery_worker.py
```

Comment:
You probably don't have any of the extra files because your application isn't flaskbb I also doubt you have the elastic beanstack extensions. but maybe that's useful